nerdlandbot/helpers/TranslationHelper.py

```diff
@@ -12,3 +14,23 @@ async def get_culture_from_context(ctx: commands.Context) -> str:
     guild_id = ctx.guild.id
     guild = await get_guild_data(guild_id)
     return guild.culture
+
+def foemp_or_schat():
+    # if it's weekend (5 or 6) the bot will say schatje instead of foemp.
+    adjective = "foemp"
+    if datetime.now().weekday() == SATURDAY or datetime.now().weekday() == SUNDAY:
```

Comment:
For future reference: you can also use `if datetime.now().weekday() in [SATURDAY, SUNDAY]:` here and on line 28. Especially useful if you have more than 2 options.